src/braket/ahs/analog_hamiltonian_simulation.py

```diff
@@ -74,7 +74,7 @@ def _hamiltonian_to_ir(self) -> ir.Hamiltonian:
             terms[term_type].append(term_ir)
         return ir.Hamiltonian(
             drivingFields=terms[AnalogHamiltonianSimulation.DRIVING_FIELDS_PROPERTY],
-            shiftingFields=terms[AnalogHamiltonianSimulation.SHIFTING_FIELDS_PROPERTY],
+            shiftingFields=terms[AnalogHamiltonianSimulation.LOCAL_DETUNING_PROPERTY],
```

Comment:
Does `shiftingFields` need to change to `localDetuning` or is local detuning a type of shifting field?

Reply:
Fixed!